src/tqec/generation/circuit_test.py

```diff
@@ -78,22 +78,54 @@ def test_generate_circuit():
     generate_qubits = [(q.row, q.col) for q in generated_circuit.all_qubits()]
     target_qubits = [
         (7, 3),
```

Comment:
May want to move this to a `test-inputs` file for reusability/readability

Reply:
I think its fine for now, the test is not very expressive anyway.

Reply:
I will try to replace that test by more "atomic"/unitary tests in #130 if it turns out that it makes sense and is doable. For the moment, I agree that this test can be kept like that.